files/default-vars-tower.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
What is this file for? I don't see it used anywhere in here ... could it be eliminated?